kfdefs/overlays/osc/osc-cl1/superset/kfdef.yaml

```diff
@@ -11,13 +11,15 @@ spec:
       name: odh-common
     - kustomizeConfig:
         parameters:
-          - name: SUPERSET_IMAGE
-            value: quay.io/opendatahub/superset:1.3.0
+          - name: superset_secret
+            value: "superset-custom"
+          - name: superset_db_secret
+            value: "superset-db-custom"
         repoRef:
           name: manifests
           path: superset
       name: superset
   repos:
     - name: manifests
-      uri: https://github.com/operate-first/odh-manifests/tarball/osc-cl1-v1.1.0
+      uri: https://github.com/operate-first/odh-manifests/tarball/osc-cl1-v1.1.2
   version: v1.1.0
```

Comment:
Should this version still point to `v1.1.0`? It did for #1718 ([see diff](https://github.com/operate-first/apps/pull/1718/files#diff-25b2fffcc8e4572e056fa079fb633885b4351c18548322ea97036f46dfda02d9R25)) so I left it as is. Although this will appear in all relevant PRs updating kfdefs in the process of upgrading ODH to `v1.1.2`.

Reply:
Turns out that the `spec.version` property of the kfdef schema refers to the version for the kubeflow operator ([see docs](https://www.kubeflow.org/docs/distributions/operator/introduction/#current-tested-operators-and-pre-built-images). Resolving this.